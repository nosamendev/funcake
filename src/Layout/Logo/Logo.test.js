import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import Logo from './Logo';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter});

let wrapper;
beforeEach(() => {
    wrapper = shallow(<Logo />);
});

describe('Render Logo', () => {
    
    it('render correctly Logo component', () => {  
        //const LogoComponent = renderer.create(<Logo />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });

    it('should render a link', () => {
        expect(wrapper.find(Link)).toHaveLength(1);
    })
});