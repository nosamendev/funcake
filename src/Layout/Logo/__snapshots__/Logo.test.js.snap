// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Logo render correctly Logo component 1`] = `
<Link
  className="logo"
  to="/"
/>
`;
